extensions/gitpod-web/package.json

```diff
@@ -1,510 +1,557 @@
 {
-  "name": "gitpod-web",
```

Comment:
could you revert this formatting change or put in a separate commit it creates noise to review :pray:

Reply:
I will revert and use **save without format** later